README_zh-CN.md

```diff
@@ -1,7 +1,7 @@
 <div align="center">
-  <img src="resources/lmdeploy-logo.svg" width="450"/>
+  <img src="docs/en/_static/image/lmdeploy-logo.svg" width="450"/>
 
-[![docs](https://img.shields.io/badge/docs-latest-blue)](https://lmdeploy-zh-cn.readthedocs.io/zh_CN/latest/)
+[![docs](https://img.shields.io/badge/docs-latest-blue)](https://lmdeploy-zh-cn.readthedocs.io/latest/)
```

Comment:
may change to `https://lmdeploy.readthedocs.io/zh_CN/latest/`

Reply:
sure